docs/layers/geojson-layer.md

```diff
@@ -80,69 +43,111 @@ Whether to draw an outline around polygons (solid fill). Note that
 for complex polygons, both the outer polygon as well the outlines of
 any holes will be drawn.
 
-
 #### `extruded` (Boolean, optional)
 
-Extrude Polygon and MultiPolygon features in 3D if true, using the
-`getElevation` accessor.
+Extrude Polygon and MultiPolygon features along the z-axis if set to
+true. The height of the drawn features is obtained using the `getElevation` accessor.
 
+- Default: `false`
 
-#### `lightSettings` (Object, optional) **EXPERIMENTAL**
-
-This is an object that contains light settings for extruded polygons.
-Be aware that this prop will likely be changed in a future version of deck.gl.
+#### `wireframe` (Boolean, optional)
 
+- Default: `false`
 
-## Callbacks
+Whether to generate a line wireframe of the hexagon. The outline will have
+"horizontal" lines closing the top and bottom polygons and a vertical line
+(a "strut") for each vertex on the polygon.
 
-#### `onHover` (Function, optional)
```

Comment:
About onHover, onClick  I think the key things we were trying to say is that the callbacks return a 'feature'. Since the GeoJson is not a standard array format. It could just be a bullet in the remarks.